Text/Chapter Text/Chapter 5.txt

```diff
@@ -0,0 +1,73 @@
+
+## Ch5OpenningText
+[OpenMidLeft][LoadVedge]
+What? There's no bridge? Looks[NL]
+like we have a fight on our[A][NL]
+hands regardless.[A][X]
+
+## Ch5VillageText
+[OpenMidRight][LoadCurrent][OpenMidLeft][LoadXimena]
+Who are you? I bet you're looking for[NL]
+a way across, aren't you? Sorry, but[A][NL]
+I don't think you'll be able to find[NL]
+it here... or anywhere within three[A][NL]
+days distance upstream or downstream.[NL]
+See, my ancestors here were persecuted[A][NL]
+relentlessly by mauraders across the[NL]
+stream, [.]and-[A]
+[OpenMidRight]Look. I don't care. Just[NL]
+tell us the best way across.[A][NL]
+It's very important.[A][CloseSpeech]
+[OpenMidLeft][MoveLeft][OpenFarLeft][LoadFace][0x5F][0x01]
+Important you say? Go to the temple[NL]
+just east of here. Just inside, you[A][NL]
```

Comment:
So.... the temple is to the west of the village. Is this an intentional thing or?